tools/metrics/gauges.go

```diff
@@ -0,0 +1,264 @@
+package main
+
+import (
+	"net/url"
+	"time"
+
+	cfclient "github.com/cloudfoundry-community/go-cfclient"
+)
+
+func QuotaGauge(c *Client, interval time.Duration) MetricReadCloser {
+	return NewMetricPoller(interval, func(w MetricWriter) error {
+		orgs, err := c.cf.ListOrgs()
+		if err != nil {
+			return err
+		}
+
+		reservedMemory := 0
+		reservedServices := 0
+		allocatedMemory := 0
+		allocatedServices := 0
+		reservedRoutes := 0
+
+		for _, org := range orgs {
+			quota, err := org.Quota()
+			if err != nil {
+				return err
+			}
+			reservedMemory += quota.MemoryLimit
+			reservedServices += quota.TotalServices
+			reservedRoutes += quota.TotalRoutes
+		}
+
+		apps, err := c.cf.ListApps()
+		if err != nil {
+			return err
+		}
+		for _, app := range apps {
+			allocatedMemory += (app.Memory * app.Instances)
+		}
+
+		allocatedServices, err = c.CountServiceInstances()
+		if err != nil {
+			return err
+		}
+
+		return w.WriteMetrics([]Metric{
+			{
+				Kind:  Gauge,
+				Time:  time.Now(),
+				Name:  "quota.services.reserved", // number of services reserved by quotas
+				Value: float64(reservedServices),
+			},
+			{
+				Kind:  Gauge,
+				Time:  time.Now(),
+				Name:  "quota.services.allocated", // number of services in use
+				Value: float64(allocatedServices),
+			},
+			{
+				Kind:  Gauge,
+				Time:  time.Now(),
+				Name:  "quota.memory.reserved", // memory reserved by org quotas
+				Value: float64(reservedMemory),
+			},
+			{
+				Kind:  Gauge,
+				Time:  time.Now(),
+				Name:  "quota.memory.allocated", // memory allocated to apps
+				Value: float64(allocatedMemory),
+			},
+			{
+				Kind:  Gauge,
+				Time:  time.Now(),
+				Name:  "quota.routes.reserved", // number of routes reserved
+				Value: float64(reservedRoutes),
+			},
+		})
+	})
+}
+
+func UserCountGauge(c *Client, interval time.Duration) MetricReadCloser {
```

Comment:
Yeah good point... I'm not sure if we care about users not-assigned to orgs since we're interested in tennant usage but you're right it's worth mentioning in the README.